sources/representation/qt/mainform.cpp

```diff
@@ -179,19 +184,47 @@ void MainForm::startGameLoop(int id, QString map)
     QElapsedTimer process_performance;
     qint64 max_process_time = 0;
 
+    int delta_time = 0;
+    float time_per_frame_ms = 1000 / fps_cap_;
+    float sleep_time = 0;
+    QElapsedTimer delta;
+    delta.start();
     while (true)
     {
-        ++fps_counter;
-
         process_performance.start();
-        GetRepresentation().Process();
-        GetScreen().Swap();
+        if(fps_cap_)
+        {
+            GetRepresentation().Process();
+            GetScreen().Swap();
+            ++fps_counter;
```

Comment:
It is not needed to restart timer after, because `process_performance.start();` do that just before processing.